File: Quan_ly_cong_ty_C/PhongBan.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Quan_ly_cong_ty_C
{
    public class PhongBan
    {
        public string MaPB { get; set; } // primary key 
        public string TenPB { get; set; }
        public PhongBan() {
            this.DsNhanVien = new List<NhanVien>();
            this.DsDuAn = new List<DuAn>();
            this.DsDiaDiem = new List<DiaDiem_Phong>();
        }
        public PhongBan(String MaPB, String TenPB, ICollection<NhanVien> DsNhanVien,ICollection<DuAn> DsDuAn, ICollection<DiaDiem_Phong> DsDiaDiem)
        {
            this.MaPB = MaPB;
            this.TenPB = TenPB;
            this.DsNhanVien = DsNhanVien;
            this.DsDuAn = DsDuAn;
            this.DsDiaDiem = DsDiaDiem;
        }
        public PhongBan(PhongBan pb)
        {
            this.MaPB = pb.MaPB;
            this.TenPB = pb.TenPB;
            this.DsNhanVien = pb.DsNhanVien;
            this.DsDuAn = pb.DsDuAn;
            this.DsDiaDiem = pb.DsDiaDiem;
        }
        public ICollection<NhanVien> DsNhanVien { get; set; }
        public ICollection<DuAn> DsDuAn { get; set; }
        public ICollection<DiaDiem_Phong> DsDiaDiem { get; set; }

    }
}

File: QuanLyCongTy Java/Quanli/src/congty/PhongBan.java
package congty;

import java.util.ArrayList;
import java.util.List;

public class PhongBan
{
	public PhongBan() { }
	public PhongBan(String MaPB,String TenPB,List<NhanVien> DsNhanVien,List<DuAn> DsDuAn,List<DiaDiem_Phong> DsDiaDiem)
	{
		this.MaPB = MaPB;
		this.TenPB = TenPB;
		this.DsNhanVien = DsNhanVien;
		this.DsDuAn = DsDuAn;
		this.DsDiaDiem = DsDiaDiem;
	}
	public PhongBan(PhongBan pb) {
		this.MaPB = pb.MaPB;
		this.TenPB = pb.TenPB;
		this.DsNhanVien = pb.DsNhanVien;
		this.DsDuAn = pb.DsDuAn;
		this.DsDiaDiem = pb.DsDiaDiem;
	}
	public String MaPB;
	public String TenPB;
	List<NhanVien> DsNhanVien = new ArrayList<NhanVien>();
	List<DuAn> DsDuAn = new ArrayList<DuAn>();
	List<DiaDiem_Phong> DsDiaDiem = new ArrayList<DiaDiem_Phong>();
	
	public String getMaPB() {
		return MaPB;
	}
	public void setMaPB(String maPB) {
		MaPB = maPB;
	}
	public String getTenPB() {
		return TenPB;
	}
	public void setTenPB(String tenPB) {
		TenPB=tenPB;
	}
	public List<NhanVien> getDsNhanVien(){
		return DsNhanVien;
	}
	public void setDsNhanVien(List<NhanVien> dsNhanVien) {
		DsNhanVien = dsNhanVien;
	}
	public List<DuAn> getDsDuAn(){
		return DsDuAn;
	}
	public void setDsDuAn(List<DuAn> dsDuAn) {
		DsDuAn = dsDuAn;
	}
	public List<DiaDiem_Phong> getDsDiaDiem(){
		return DsDiaDiem;
	}
	public void setDsDiaDiem(List<DiaDiem_Phong> dsDiaDiem) {
		DsDiaDiem = dsDiaDiem;
	}
}
